### ISSUE TITLE
Extract feature toggle library from bundle

### ISSUE THREAD

**Issue by sagikazarmark** [elbcoast/ECNFeatureToggleBundle#13]
Possible competitor: https://github.com/qandidate-labs/qandidate-toggle-bundle

Downsides: It is unnecessarily complex.

**Comment by elbcoast** [comment]
Absolutely. Splitting this up into three different packages makes sense: One for the library, one for twig related stuff and one symfony specific bundle which will also contain the annotation logic for the controllers.
